### PR TITLE
FLW-86 API Key "ProviderServiceMapId" Added

### PR DESCRIPTION
**Pull request by sushant-bizbrolly**
## 📋 Description

JIRA ID: 

Please provide a summary of the change and the motivation behind it. Include relevant context and details.

---

## ✅ Type of Change

- [ ] 🐞 **Bug fix** (non-breaking change which resolves an issue)
- [ ] ✨ **New feature** (non-breaking change which adds functionality)
- [ ] 🔥 **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🛠 **Refactor** (change that is neither a fix nor a new feature)
- [ ] ⚙️ **Config change** (configuration file or build script updates)
- [ ] 📚 **Documentation** (updates to docs or readme)
- [ ] 🧪 **Tests** (adding new or updating existing tests)
- [ ] 🎨 **UI/UX** (changes that affect the user interface)
- [ ] 🚀 **Performance** (improves performance)
- [ ] 🧹 **Chore** (miscellaneous changes that don't modify src or test files)

---

## ℹ️ Additional Information

Please describe how the changes were tested, and include any relevant screenshots, logs, or other information that provides additional context.

## Summary by CodeRabbit

- **New Features**
	- Enhanced beneficiary and household data handling by integrating user-specific information.
	- Added user data integration in CBAC submission process.
- **Bug Fixes**
	- Improved error handling for server responses in beneficiary data operations.
- **Chores**
	- Cleaned up unused properties in user-related data classes.

### CODE REVIEW

File: app/src/main/java/org/piramalswasthya/sakhi/model/Household.kt [hunk 1]
```diff
@@ -80,11 +80,11 @@ data class HouseholdCache(
 ) : FormDataModel {
 
 
-    fun asNetworkModel(): HouseholdNetwork {
+    fun asNetworkModel(user: User): HouseholdNetwork {
         return HouseholdNetwork(
             Countyid = locationRecord.country.id,
             Processed = processed,
-//            ProviderServiceMapID = userCache.serviceMapId,
+            ProviderServiceMapID = user.serviceMapId,
```
[the remark this answers is not in the document]
Comment (sushant-bizbrolly) [reply app/src/main/java/org/piramalswasthya/sakhi/model/Household.kt:87]: Handled

**Comment by drtechie** [comment]
@sushant-bizbrolly can we have some consistency on the attributes.
We are using PascalCase, camelCase and snake_case.

**Comment by drtechie** [comment]
Can you also add JIRA details?

**Comment by sushant-bizbrolly** [comment]
> Can you also add JIRA details?

@drtechie , No JIRA ticket associated with it. @madhavaramu83 conveyed verbally.

**Review by NikhilFegade — APPROVED** [review]

**Comment by sushant-bizbrolly** [comment]
> @sushant-bizbrolly can we have some consistency on the attributes. We are using PascalCase, camelCase and snake_case.

Done @drtechie

**Review by drtechie — APPROVED** [review]

**Pull request merged**